### DISCUSSION TITLE
Eform 17 - Error OPT-060-Lot

### DISCUSSION THREAD

**Discussion by lauralu1993** [OP-TED/eForms-SDK#740] [Q&A] [answered]
after filling out Eform 17, the following error occurs during the validation:

<img width="578" alt="image" src="https://github.com/OP-TED/eForms-SDK/assets/142924847/b2672ec4-5da8-4d62-a46a-5ff01a089cce">

the field “OPT-060-Lot”  does not appear in any section. Therefore, it is not possible to resolve the error.
nr. procedure = cdd50973-4abc-48bb-9870-f2a12f223374

<?xml version="1.0" encoding="UTF-8"?>
<ContractNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractNotice-2"
                 xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
                 xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2"
                 xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1"
                 xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1"
                 xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1"
                 xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2"
                 >
   <ext:UBLExtensions>
      <ext:UBLExtension>
         <ext:ExtensionContent>
            <efext:EformsExtension>
               <efac:NoticeSubType>
                  <cbc:SubTypeCode listName="notice-subtype">17</cbc:SubTypeCode>
               </efac:NoticeSubType>
               <efac:Organizations>
                  <efac:Organization>
                     <efac:Company>
                        <cac:PartyIdentification>
                           <cbc:ID>ORG-0001</cbc:ID>
                        </cac:PartyIdentification>
                        <cac:PartyName>
                           <cbc:Name languageID="ITA">P1_17 - Bando di gara — direttiva settoriale, regime ordinario</cbc:Name>
                        </cac:PartyName>
                        <cac:PostalAddress>
                           <cbc:CityName>BT-8</cbc:CityName>
                           <cac:Country>
                              <cbc:IdentificationCode listName="country">ITA</cbc:IdentificationCode>
                           </cac:Country>
                        </cac:PostalAddress>
                        <cac:PartyLegalEntity>
                           <cbc:CompanyID>BT-5</cbc:CompanyID>
                        </cac:PartyLegalEntity>
                        <cac:Contact>
                           <cbc:Telephone>081223322</cbc:Telephone>
                           <cbc:ElectronicMail>mail@gmail.com</cbc:ElectronicMail>
                        </cac:Contact>
                     </efac:Company>
                  </efac:Organization>
               </efac:Organizations>
            </efext:EformsExtension>
         </ext:ExtensionContent>
      </ext:UBLExtension>
   </ext:UBLExtensions>
   <cbc:UBLVersionID>2.3</cbc:UBLVersionID>
   <cbc:CustomizationID>eforms-sdk-1.9</cbc:CustomizationID>
   <cbc:ID schemeName="notice-id">cdd50973-4abc-48bb-9870-f2a12f223374</cbc:ID>
   <cbc:ContractFolderID>2818efa4-58c6-4aa9-b509-4a8d997b5006</cbc:ContractFolderID>
   <cbc:IssueDate>2023-10-26Z</cbc:IssueDate>
   <cbc:IssueTime>10:35:23Z</cbc:IssueTime>
   <cbc:VersionID>01</cbc:VersionID>
   <cbc:RegulatoryDomain>32014L0024</cbc:RegulatoryDomain>
   <cbc:NoticeTypeCode listName="competition">cn-standard</cbc:NoticeTypeCode>
   <cbc:NoticeLanguageCode listName="language">ITA</cbc:NoticeLanguageCode>
   <cac:ContractingParty>
      <cac:ContractingActivity>
         <cbc:ActivityTypeCode listName="entity-activity">electricity</cbc:ActivityTypeCode>
      </cac:ContractingActivity>
      <cac:Party>
         <cac:PartyIdentification>
            <cbc:ID>ORG-0001</cbc:ID>
         </cac:PartyIdentification>
      </cac:Party>
   </cac:ContractingParty>
   <cac:TenderingProcess>
      <cbc:ProcedureCode listName="procurement-procedure-type">open</cbc:ProcedureCode>
   </cac:TenderingProcess>
   <cac:ProcurementProject>
      <cbc:Name languageID="ITA">BT-21</cbc:Name>
      <cbc:Description languageID="ITA">BT-88</cbc:Description>
      <cbc:ProcurementTypeCode listName="contract-nature">supplies</cbc:ProcurementTypeCode>
      <cac:MainCommodityClassification>
         <cbc:ItemClassificationCode listName="cpv">39160000</cbc:ItemClassificationCode>
      </cac:MainCommodityClassification>
   </cac:ProcurementProject>
   <cac:ProcurementProjectLot>
      <cbc:ID schemeName="Lot">LOT-0001</cbc:ID>
      <cac:TenderingTerms>
         <ext:UBLExtensions>
            <ext:UBLExtension>
               <ext:ExtensionContent>
                  <efext:EformsExtension>
                     <efac:SelectionCriteria>
                        <cbc:CriterionTypeCode listName="selection-criterion">ef-stand</cbc:CriterionTypeCode>
                     </efac:SelectionCriteria>
                  </efext:EformsExtension>
               </ext:ExtensionContent>
            </ext:UBLExtension>
         </ext:UBLExtensions>
         <cac:RequiredFinancialGuarantee>
            <cbc:GuaranteeTypeCode listName="tender-guarantee-required">false</cbc:GuaranteeTypeCode>
         </cac:RequiredFinancialGuarantee>
         <cac:CallForTendersDocumentReference>
            <cbc:ID>_DEFAULT_VALUE_CHANGE_ME_</cbc:ID>
            <cbc:DocumentType>restricted-document</cbc:DocumentType>
            <cac:Attachment>
               <cac:ExternalReference>
                  <cbc:URI>https://www.service.com</cbc:URI>
               </cac:ExternalReference>
            </cac:Attachment>
         </cac:CallForTendersDocumentReference>
         <cac:PaymentTerms>
            <cbc:Note languageID="ITA">BT 77</cbc:Note>
         </cac:PaymentTerms>
         <cac:TendererQualificationRequest>
            <cbc:CompanyLegalFormCode listName="required">false</cbc:CompanyLegalFormCode>
         </cac:TendererQualificationRequest>
         <cac:TendererQualificationRequest>
            <cac:SpecificTendererRequirement>
               <cbc:TendererRequirementTypeCode listName="reserved-procurement">none</cbc:TendererRequirementTypeCode>
            </cac:SpecificTendererRequirement>
         </cac:TendererQualificationRequest>
         <cac:ContractExecutionRequirement>
            <cbc:ExecutionRequirementCode listName="reserved-execution">no</cbc:ExecutionRequirementCode>
         </cac:ContractExecutionRequirement>
         <cac:ContractExecutionRequirement>
            <cbc:ExecutionRequirementCode listName="ecatalog-submission">not-allowed</cbc:ExecutionRequirementCode>
         </cac:ContractExecutionRequirement>
         <cac:AdditionalInformationParty>
            <cac:PartyIdentification>
               <cbc:ID>ORG-0001</cbc:ID>
            </cac:PartyIdentification>
         </cac:AdditionalInformationParty>
         <cac:TenderRecipientParty>
            <cbc:EndpointID>https://www.service.com</cbc:EndpointID>
         </cac:TenderRecipientParty>
         <cac:AppealTerms>
            <cac:AppealReceiverParty>
               <cac:PartyIdentification>
                  <cbc:ID>ORG-0001</cbc:ID>
               </cac:PartyIdentification>
            </cac:AppealReceiverParty>
         </cac:AppealTerms>
         <cac:Language>
            <cbc:ID>ITA</cbc:ID>
         </cac:Language>
      </cac:TenderingTerms>
      <cac:TenderingProcess>
         <cbc:SubmissionMethodCode listName="esubmission">allowed</cbc:SubmissionMethodCode>
         <cbc:GovernmentAgreementConstraintIndicator>false</cbc:GovernmentAgreementConstraintIndicator>
         <cac:TenderSubmissionDeadlinePeriod>
            <cbc:EndDate>2023-10-31+01:00</cbc:EndDate>
            <cbc:EndTime>12:00:00+01:00</cbc:EndTime>
         </cac:TenderSubmissionDeadlinePeriod>
         <cac:AuctionTerms>
            <cbc:AuctionConstraintIndicator>false</cbc:AuctionConstraintIndicator>
         </cac:AuctionTerms>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="framework-agreement">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="dps-usage">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
      </cac:TenderingProcess>
      <cac:ProcurementProject>
         <cbc:ID>BT-1</cbc:ID>
         <cbc:Name languageID="ITA">BT-2</cbc:Name>
         <cbc:Description languageID="ITA">BT-3</cbc:Description>
         <cbc:ProcurementTypeCode listName="contract-nature">supplies</cbc:ProcurementTypeCode>
         <cac:MainCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">39160000</cbc:ItemClassificationCode>
         </cac:MainCommodityClassification>
         <cac:ContractExtension>
            <cbc:MaximumNumberNumeric>33</cbc:MaximumNumberNumeric>
         </cac:ContractExtension>
      </cac:ProcurementProject>
   </cac:ProcurementProjectLot>
   <cac:ProcurementProjectLot>
      <cbc:ID schemeName="Lot">LOT-0002</cbc:ID>
      <cac:TenderingTerms>
         <ext:UBLExtensions>
            <ext:UBLExtension>
               <ext:ExtensionContent>
                  <efext:EformsExtension>
                     <efac:SelectionCriteria>
                        <cbc:CriterionTypeCode listName="selection-criterion">sui-act</cbc:CriterionTypeCode>
                     </efac:SelectionCriteria>
                  </efext:EformsExtension>
               </ext:ExtensionContent>
            </ext:UBLExtension>
         </ext:UBLExtensions>
         <cac:RequiredFinancialGuarantee>
            <cbc:GuaranteeTypeCode listName="tender-guarantee-required">false</cbc:GuaranteeTypeCode>
         </cac:RequiredFinancialGuarantee>
         <cac:CallForTendersDocumentReference>
            <cbc:ID>_DEFAULT_VALUE_CHANGE_ME_</cbc:ID>
            <cbc:DocumentType>restricted-document</cbc:DocumentType>
            <cac:Attachment>
               <cac:ExternalReference>
                  <cbc:URI>https://www.service.com</cbc:URI>
               </cac:ExternalReference>
            </cac:Attachment>
         </cac:CallForTendersDocumentReference>
         <cac:PaymentTerms>
            <cbc:Note languageID="ITA">BT-44</cbc:Note>
         </cac:PaymentTerms>
         <cac:TendererQualificationRequest>
            <cbc:CompanyLegalFormCode listName="required">false</cbc:CompanyLegalFormCode>
         </cac:TendererQualificationRequest>
         <cac:TendererQualificationRequest>
            <cac:SpecificTendererRequirement>
               <cbc:TendererRequirementTypeCode listName="reserved-procurement">none</cbc:TendererRequirementTypeCode>
            </cac:SpecificTendererRequirement>
         </cac:TendererQualificationRequest>
         <cac:ContractExecutionRequirement>
            <cbc:ExecutionRequirementCode listName="reserved-execution">no</cbc:ExecutionRequirementCode>
         </cac:ContractExecutionRequirement>
         <cac:ContractExecutionRequirement>
            <cbc:ExecutionRequirementCode listName="ecatalog-submission">not-allowed</cbc:ExecutionRequirementCode>
         </cac:ContractExecutionRequirement>
         <cac:AdditionalInformationParty>
            <cac:PartyIdentification>
               <cbc:ID>ORG-0001</cbc:ID>
            </cac:PartyIdentification>
         </cac:AdditionalInformationParty>
         <cac:TenderRecipientParty>
            <cbc:EndpointID>http://www.service.com</cbc:EndpointID>
         </cac:TenderRecipientParty>
         <cac:AppealTerms>
            <cac:AppealReceiverParty>
               <cac:PartyIdentification>
                  <cbc:ID>ORG-0001</cbc:ID>
               </cac:PartyIdentification>
            </cac:AppealReceiverParty>
         </cac:AppealTerms>
         <cac:Language>
            <cbc:ID>ITA</cbc:ID>
         </cac:Language>
      </cac:TenderingTerms>
      <cac:TenderingProcess>
         <cbc:SubmissionMethodCode listName="esubmission">allowed</cbc:SubmissionMethodCode>
         <cbc:GovernmentAgreementConstraintIndicator>false</cbc:GovernmentAgreementConstraintIndicator>
         <cac:TenderSubmissionDeadlinePeriod>
            <cbc:EndDate>2023-10-31+01:00</cbc:EndDate>
            <cbc:EndTime>12:00:00+01:00</cbc:EndTime>
         </cac:TenderSubmissionDeadlinePeriod>
         <cac:AuctionTerms>
            <cbc:AuctionConstraintIndicator>false</cbc:AuctionConstraintIndicator>
         </cac:AuctionTerms>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="framework-agreement">none</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
         <cac:ContractingSystem>
            <cbc:ContractingSystemTypeCode listName="dps-usage">dps-list</cbc:ContractingSystemTypeCode>
         </cac:ContractingSystem>
      </cac:TenderingProcess>
      <cac:ProcurementProject>
         <cbc:ID>BT-2</cbc:ID>
         <cbc:Name languageID="ITA">BT-3</cbc:Name>
         <cbc:Description languageID="ITA">BT-4</cbc:Description>
         <cbc:ProcurementTypeCode listName="contract-nature">supplies</cbc:ProcurementTypeCode>
         <cac:MainCommodityClassification>
            <cbc:ItemClassificationCode listName="cpv">39160000</cbc:ItemClassificationCode>
         </cac:MainCommodityClassification>
         <cac:ContractExtension>
            <cbc:MaximumNumberNumeric>66556</cbc:MaximumNumberNumeric>
         </cac:ContractExtension>
      </cac:ProcurementProject>
   </cac:ProcurementProjectLot>
</ContractNotice>

**Comment by YvesJo** [comment, marked as answer]
OPT-060-Lot is a non displayed technical field managed by the application, it is associated to the field BT-70-Lot and provides the necessary semantic. It is only generated in the XML when the associated text (description of the terms of performance) is present.
The second lot in the notice is missing the Terms of Performance description (BT-70).
